### PR TITLE
Feature: add sessions lookml

### PR DESCRIPTION
**Pull request by axelazaid**
* added heap traffic summary lookml for `fct_web_sessions`

### CODE REVIEW

File: lookml/marketing.model.lkml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+connection: "redshift"
```
Comment (jthandy) [inline lookml/marketing.model.lkml]: you can't specify a connection in a shared project :D change this to something like:
```
# connection: "enter your connection here"
```

Comment (jthandy) [reply lookml/marketing.model.lkml]: (the # is a lookml comment)

File: lookml/web_sessions.view.lkml [hunk 2]
```diff
@@ -0,0 +1,320 @@
+view: web_sessions {
+  sql_table_name: {{_user_attributes['dbt_schema']}}.fct_web_sessions ;;
```
Comment (jthandy) [inline lookml/web_sessions.view.lkml]: `fct_web_sessions` actually isn't something that exists in the package. I think we just need to point this to `heap_sessions_xf`, right?

File: lookml/web_sessions.view.lkml [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+
+  measure: bounce_rate {
+    type: number
+    sql: ${bounced_sessions}::float / nullif(${sessions}, 0) ;;
+    value_format_name: percent_1
+  }
+
+}
+
+view: web_sessions_ad_performance {
```
Comment (jthandy) [inline lookml/web_sessions.view.lkml]: this is all custom stuff for OV--shouldn't be included here :D

**Comment by jthandy** [comment]
Nice work on this! Small changes then we can merge :D

File: lookml/marketing.model.lkml (hunk 1 above)
Comment (axelazaid) [reply lookml/marketing.model.lkml]: got it! :)

File: lookml/web_sessions.view.lkml (hunk 3 above)
Comment (axelazaid) [reply lookml/web_sessions.view.lkml]: i'm confused about this. i'm using some of these measure to build the charts. what exactly are you referring to that's OV-specific?

Comment (jthandy) [reply lookml/web_sessions.view.lkml]: just the things below this line. `web_sessions_ad_performance` is a view that _extends_ the `web_sessions` view. that stuff is custom. the `web_sessions` view is all generic and should be a part of the package.

please let me know if you'd like to talk this through!

Comment (axelazaid) [reply lookml/web_sessions.view.lkml]: 👍

**Pull request merged**